<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RechercheInteractions</class>
 <widget class="QWidget" name="RechercheInteractions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="interactionsTableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>621</width>
     <height>351</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Contenu</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date d'ajout</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>621</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Recherche interactions</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>513</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="dAjoutLabel">
       <property name="text">
        <string>Date d'ajout</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="dAjoutCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dIntervalleInterRechercheLabel">
       <property name="text">
        <string>Interavalle date de création</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDateEdit" name="dateAjoutEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="dIntervalleInterCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="labelDebutInter">
         <property name="text">
          <string>Début</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dDebutInteraction">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelFinInter">
         <property name="text">
          <string>Fin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dFinInteraction">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="rechercheFiltreInterButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="suppressionFiltreInterButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer filtre</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="erreurRechercheLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>621</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
